### ISSUE TITLE
Keyboard Shuttle

### ISSUE THREAD

**Issue by TheDcoder** [PeterCxy/Shelter#137]
I recently switched to Android 10 and I found out that keyboards are now separated for work profile. Prior to this, there used to be only a single keyboard for all profiles. This was a convenient feature, especially for users with specialized keyboards (like KeePass2Android) who want to use the same keyboard with sheltered apps.

Is it technically possible to develop something like a "keyboard shuttle"? Or is there any other way to use the same personal profile keyboard with work profile apps?

**Comment by selimovd** [comment]
I agree with @TheDcoder , I also use 4 languages and I don't really want to install them twice.
Is it possible to use one keyboard with the normal and with the work profile?
